.github/workflows/update-deps.yml

```diff
@@ -0,0 +1,32 @@
+name: Update dependencies
```

Comment:
maybe extend the name and file name to clarify that it is about "internal" dependencies?